mne/time_frequency/tfr.py

```diff
@@ -2064,12 +2176,30 @@ def __init__(self, info, data, times, freqs, comment=None, method=None,
         if events is None:
             n_epochs = len(data)
             events = _gen_events(n_epochs)
+        if selection is None:
+            n_epochs = len(data)
+            selection = np.arange(n_epochs)
+        if drop_log is None:
+            drop_log = tuple(
+                () if k in selection else ('IGNORED',)
+                for k in range(max(len(events),
+                               max(selection) + 1)))
+        else:
+            drop_log = drop_log
+        # check consistency:
+        assert len(selection) == len(events)
+        assert len(drop_log) >= len(events)
+        assert len(selection) == sum(
+            (len(dl) == 0 for dl in drop_log))
+        # TODO Check for consistency with .metadata?
```

Comment:
is this cruft or a valid `TODO`?

Reply:
I guess this can be ignored. It was a note to myself to check back whether you (mne maintainers) want to see such checks. 
I looked it up, and there are also no consistency checks against (the shape of) `metadata` for other properties of the TFR classes (e.g., `events`) or for `selection` in the `(Base)Epochs` class. So I assume this is/was not seen necessary. 
```suggestion
```